pages.zh/osx/port.md

```diff
@@ -1,6 +1,6 @@
 # port
 
-> MacOS 包管理器软件（类似 brew).
+> macOS 包管理器软件（类似 brew).
```

Comment:
What if this is the star of the sentence?

Reply:
I'm not exactly sure what you mean.  The [English page](https://github.com/tldr-pages/tldr/blob/6a058b56fe090debff720d06196c02199ea5644c/pages/osx/port.md) says this:

> Package manager for macOS.

Google translates that Chinese to `macOS package manager software (similar to brew).`

The term "MacOS" isn't correct anymore.  Apple has replaced it with "macOS" (to my knowledge).

Reply:
I mean a sentence should start with an upper case letter. But maybe I'm wrong

Reply:
I don't think that applies to things like product names. On its [wikipedia article](https://en.wikipedia.org/wiki/MacOS) it's also lowercase, even at the start of the sentence.

Reply:
Here is the recommendation from [Google developer documentation style guide](https://developers.google.com/style/capitalization):

> If an official name begins with a lowercase letter, then put it in lowercase even at the start of a sentence. But it's better to revise the sentence to avoid putting a lowercase word at the start, if possible.

Reply:
I could find a similar reference in the `The Chicago Manual of Style` and the `Oxford University Style Guide`, but I only spent a few seconds searching... :innocent:

Reply:
Then let's keep it.

>>But it's better to revise the sentence to avoid putting a lowercase word at the start, if possible.

Exactly what I would say. 👍🏻

Reply:
this reminds me of the comments on my `sic` PR, where I got pushed into using uppercase..

Reply:
This was an unusual, but very deliberate design choice by Apple to call their operating system macOS. It doesn't make proper grammatical sense, but it's the correct nonetheless.